src/index.ts

```diff
@@ -73,6 +74,7 @@ export { CustomEmojiId } from "./types/CustomEmojiId";
 export { CustomEmojiKeyword } from "./types/CustomEmojiKeyword";
 export { CustomEmojiResponse } from "./types/CustomEmojiResponse";
 export { CustomEmojiView } from "./types/CustomEmojiView";
+export { DbUrl } from "./types/DbUrl";
```

Comment:
Should `DbUrl` and `SensitiveString` be exposed as types for this? They're both just aliases for `string` anyway.

Reply:
Its what ts-rs generates. Its not really worth it to try to rewrite the wrapper types `ts(as....`) on the back end, especially since these are just simple aliases.